mpa/modules/models/heads/cross_dataset_detector_head.py

```diff
@@ -236,4 +217,15 @@ def vfnet_to_atss_targets(self,
         ]
         label_weights = torch.cat(label_weights_list)
         bbox_weights = torch.cat(bbox_weights_list)
-        return labels_list, label_weights, bbox_targets_list, bbox_weights
+        return labels_list, label_weights, bbox_targets_list, bbox_weights, ignored_masks
+
+    def get_ignored_masks(self, img_metas, all_labels):
```

Comment:
Is this 'ignored' mask or 'valid' mask?
Actually, valid_label_mask would be better in terms of clarification.

Reply:
The ignored_masks were modified to valid_label_mask.